pandas/tseries/tools.py

```diff
@@ -237,6 +237,10 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
         integer or float number. This will be based off the origin.
         Example, with unit='ms' and origin='unix' (the default), this
         would calculate the number of milliseconds to the unix epoch start.
+
+        Warning: For float arg, precision rounding might happen. To prevent
+        unexpected behavior use a fixed-width exact type.
```

Comment:
If you mean for `unit='ns'` I don't agree. You can use `unit='s'` and have result saved up to mili seconds with rounding error from float. as in timeseries example